packages/wrangler/src/__tests__/deploy/config-diffs.test.ts

```diff
@@ -0,0 +1,186 @@
+import { getRemoteConfigDiff } from "../../deploy/config-diffs";
+import type { Config, RawConfig } from "../../config";
+
+describe("getRemoteConfigsDiff", () => {
+	it("should handle a very simple diffing scenario (no diffs, random order)", () => {
+		const { diff, onlyAdditionsIfAny } = getRemoteConfigDiff(
+			{
+				name: "silent-firefly-dbe3",
+				main: "/Users/dario/Desktop/silent-firefly-dbe3/src/index.js",
```

Comment:
Is this intended?

Reply:
mh.... kind of 😅, the value there doesn't really matter so I was thinking of just hardcoding something, but yeah `/Desktop/etc...` is super ugly, I forgot to clean it up a bit 😓

Reply:
Fixed 🙂